scripts/build-images.sh

```diff
@@ -20,13 +20,13 @@ COMMIT_SHA=${LND_VERSION}
 LND_BTC_NODE=bitcoind
 LND_LTC_NODE=litecoind
 
-# NOTE: The names specified with `-t` directly map to the service names in
-# the applicable services docker-compose file
-docker build -t sparkswap_lnd_btc ./docker/lnd --build-arg NODE=$LND_BTC_NODE --build-arg NETWORK=btc --build-arg COMMIT_SHA=$COMMIT_SHA
-docker build -t sparkswap_lnd_ltc ./docker/lnd --build-arg NODE=$LND_LTC_NODE --build-arg NETWORK=ltc --build-arg COMMIT_SHA=$COMMIT_SHA
+# NOTE: The names specified with `-t` directly map to the our service names in
+# on sparkswap dockerhub
+docker build -t sparkswap/lnd_btc:latest ./docker/lnd --build-arg NODE=$LND_BTC_NODE --build-arg NETWORK=btc --build-arg COMMIT_SHA=$COMMIT_SHA
```

Comment:
can we do this before we reference these names in the docker-compose file?

Reply:
@treygriffith can you elaborate? Do you mean building this image in broker instead?